tools/mksurfdata_map/src/mkglcmecMod.F90

```diff
@@ -555,18 +559,21 @@ subroutine mkglacier(ldomain, mapfname, datfname, ndiag, zero_out, glac_o)
      ! Error checks for domain and map consistencies
      call domain_checksame( tdomain, ldomain, tgridmap )
      
+     ! Obtain frac_dst
+     call gridmap_calc_frac_dst(tgridmap, tdomain%mask, frac_dst)
+
      ! Determine glac_o on output grid
 
-     call gridmap_areaave(tgridmap, glac_i, glac_o, nodata=0._r8)
+     call gridmap_areaave(tgridmap, glac_i, glac_o, nodata=0._r8, mask_src=tdomain%mask, frac_dst=frac_dst)
      
-     do no = 1, ldomain%ns
+     do no = 1, ns_o
         if (glac_o(no) < 1.) glac_o(no) = 0.
```

Comment:
Should I go through and add `_r8` wherever it's missing or would you rather I didn't?
@billsacks @ekluzek

Reply:
No opinion - whatever you like.

Reply:
I decided against it for now.